OWNERS

```diff
@@ -13,3 +18,5 @@ reviewers:
   - Honny1
   - mheon
   - vrothberg
+  - Luap99
+  - mtrmac
```

Comment:
Is the `reviewers` list supposed to include some, all, or none of the `approvers` list?  Looking at podman's main branch, I see one `approver` who is not also a `reviewer`, so I don't know if there's a rule here
I think both lists were previously kept sorted.